### ISSUE TITLE
Misaligned QuickSearch touch targets

### ISSUE THREAD

**Issue by noodlejetski** [BranchMetrics/sesame_issues#113]
The touch targets for each QuickSearch entry aren't aligned properly with the search icons (see the screenshot below). It often causes me to use the wrong app to search for my query, because tapping just a few pixels too far to the left of the icon launches the app to the left of the one I intended to use. The number of enabled QuickSearch apps doesn't matter, the same thing has been happening with both 3 and 7 QuickSearch apps enabled.

I'm using the latest Sesame beta (3.5.4-beta3) on Sony Xperia XZ2 Compact and XZ3, both running Android 9.0, and I've been noticing the same behaviour on both devices for a while.

![Screenshot_20190913-212217](https://user-images.githubusercontent.com/8598559/64889831-85243500-d66e-11e9-8e47-9e0b6e943b2e.png)

**Comment by philwall2** [comment]
You're right, it sucks. The truth is it's fixable but a lot more work than you'd imagine. 

Since we dynamically change the position of those icons based on how many you have ON, it messes up the standard Android layout. Our choice is to either A) have them visually aligned but the click zones askew, B) have the click zones aligned but have the icons misaligned, C) build a custom layout.

We went with A. C is on our todo list but that list never really gets smaller :)

**Comment by noodlejetski** [comment]
thanks for the reply. it's indeed surprising that it's not an easy fix. I'm looking forward to any changes in the future, as it can mess up the flow of the search action really badly.
